src/RetrescoClientImpl.php

```diff
@@ -412,7 +434,7 @@ public function poolSearchAggregations($query): array {
    * @inheritDoc
    */
   public function deserialize(string $data, string $type): object {
-    $context = ['json_decode_associative' => FALSE];
+    $context = ['json_decode_associative' => TRUE];
```

Comment:
yes, actually required since it would not work with the generated classes anymore, as it now uses array vs. formerly objects, this is because of the updated symfony serializer.

Reply:
but had very small effect on the drupal codebase, i think this is the only one https://github.com/kurier/hermes-efs/pull/2449/files#diff-83f4e4aec1c184a3e9b4edd2bedf830c6ae697b316c45752f896d94df7329354R158

Reply:
but had very small effect on the drupal codebase, i think this is the only one https://github.com/kurier/hermes-efs/pull/2449/files#diff-83f4e4aec1c184a3e9b4edd2bedf830c6ae697b316c45752f896d94df7329354R158